src/NRules/NRules.sln

```diff
@@ -19,6 +19,8 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "NRules.Json", "NRules.Json\
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "NRules.Json.Tests", "Tests\NRules.Json.Tests\NRules.Json.Tests.csproj", "{846AAC54-E7D1-4167-8D53-13B836DE7CFA}"
 EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "NRules.Testing", "NRules.Testing\NRules.Testing.csproj", "{C35D16EE-6FFD-47B4-91F9-0930A0CCD699}"
```

Comment:
@gritcsenko we'll need to create a nuspec file and integrate into build/documentation. But I can easily do it after the PR is merged